sub-042/eeg/sub-042_task-responder_run-1_eeg.vhdr

```diff
@@ -1 +0,0 @@
-../../.git/annex/objects/3k/3g/MD5E-s8385--f3282abaff034033c103a02eb0e938de.vhdr/MD5E-s8385--f3282abaff034033c103a02eb0e938de.vhdr
```

Comment:
Hey @sappelhoff and @franklin-feingold, either of you wouldn't happen to know how to suggest changes to an OpenNeuro BIDS dataset where the data files reference a `git/annex`  as in this file would you? It looks like there was a failure with the BrainVision renaming tool just for this one subject and I changed the file paths to match by hand so that readers worked and I thought it would be nice to fix for anyone else using the dataset.